flutter-app/flutter_drawing_app/starter/lib/drawing_page.dart

```diff
@@ -391,6 +391,9 @@ class _DrawingPageState extends State<DrawingPage> {
               )
           );
           setState(() {
+            if (actionsStack.isEmpty) {
+              return;
+            }
             if (this.displayImage != null) {
```

Comment:
Maybe try`if (this.displayImage != null && !actionsStack.isEmpty)` just to clean it up?

Reply:
yeah thats a way better idea, thank you